themes/doom-ir-black-theme.el

```diff
@@ -0,0 +1,164 @@
+;;; doom-ir-black-theme.el --- port of the original ir_black theme
+(require 'doom-themes)
+
+;;
+(defgroup doom-ir-black-theme nil
+  "Options for doom-themes"
+  :group 'doom-themes)
+
+(defcustom doom-ir-black-smooth-bg nil
+  "If non-nil, comments will be highlighted in more vivid colors."
+  :group 'doom-ir-black-theme
+  :type 'boolean)
+
```

Comment:
Would you mind removing this? I ask because it's relatively trivial for the end users to customize with:

```elisp
(custom-set-faces
  `(default ((t (:background "#000000")))))
```

I'm also resistant to adding variables for every minor change.

Reply:
Sorry. I was the one who added it. I added it on his request but I'll remove it then!

Reply:
No worries! I've only started (quietly) cracking down on our themes a couple days ago.

Reply:
I noticed! My email won't stop notifying me haha.